clis/teliod/src/cgi/constants.rs

```diff
@@ -0,0 +1,26 @@
+use const_format::concatcp;
+
+#[cfg(feature = "qnap")]
+mod consts {
+    use super::*;
+
+    pub const APP_DIR: &str = "/share/CACHEDEV1_DATA/.qpkg/NordSecurityMeshnet";
+}
+
+#[cfg(not(feature = "qnap"))]
+mod consts {
+    pub const APP_DIR: &str = concat!(env!("CARGO_MANIFEST_DIR"), "/target/debug");
+}
+
+use consts::*;
```

Comment:
why not directly do this? readability?

```suggestion
#[cfg(feature = "qnap")]
pub const APP_DIR: &str = "/share/CACHEDEV1_DATA/.qpkg/NordSecurityMeshnet";
#[cfg(not(feature = "qnap"))]
pub const APP_DIR: &str = concat!(env!("CARGO_MANIFEST_DIR"), "/target/debug");
```